### ISSUE TITLE
Search in date range

### ISSUE THREAD

**Issue by sirinoks** [Seneca-CDOT/telescope#2894]
Connected to #1912 

Implement an ability to search for posts within a certain date range. 

How would it look like?
 - A calendar selection (there are completed solutions we could just take and use)
 - A word format, like in discord :
 <img src="https://cdn.discordapp.com/attachments/353432193730871296/941856537415135272/Screenshot_448.png" />

We could also first implement the syntax format to make it work, and only then work on the calendar UI.

**Comment by sirinoks** [comment]
Or just steal completely how Discord does it

**Comment by cindyledev** [comment]
I was playing around with TailwindCSS and mocked a Search page, we use Material UI but we can do something like this 

![2022-02-13 12_43_23-Window](https://user-images.githubusercontent.com/36822198/153767613-7405ecbc-e702-453e-8e14-9f87c35bdb9e.png)

**Comment by AmasiaNalbandian** [comment]
this is in my branch for the searchbar....

**Comment by cindyledev** [comment]
> this is in my branch for the searchbar....

Hey @AmasiaNalbandian, sorry I didn't see you were working on the other issue until after I did this. Please disregard my comment.

**Comment by AmasiaNalbandian** [comment]
its no problem, but i am wondering if we should close this? it's part of the redesign

**Comment by cindyledev** [comment]
> its no problem, but i am wondering if we should close this? it's part of the redesign

Yes, we should close this issue and add a blurb in the other issue that it will also address the date range

**Comment by AmasiaNalbandian** [comment]
I need to post a screenshot of my progress - its really close just blocked by my limitations in React (definitely not my comfort language)

**Comment by cindyledev** [comment]
> I need to post a screenshot of my progress - its really close just blocked by my limitations in React (definitely not my comfort language)

Put up a PR! I'm sure there are React devs on the team that can help out

**Comment by tpmai22** [comment]
@AmasiaNalbandian Should I move this to 2.9 ?

**Comment by sirinoks** [comment]
@AmasiaNalbandian @cindyledev Was this solved?

**Comment by AmasiaNalbandian** [comment]
I have a [branch on my fork ](https://github.com/Seneca-CDOT/telescope/compare/master...AmasiaNalbandian:telescope:issue-2110-date)for this... If anyone is up for the challenge for this they can take it, otherwise I will look into rebasing 800+ commits... Also telescope does not run on my machine. I need up to next week to try and resolve the repository on my machine.

**Comment by sirinoks** [comment]
@AmasiaNalbandian Oh my...
